static/templates/user_profile_modal.hbs

```diff
@@ -9,6 +9,10 @@
                     <a href="/#settings/profile">
                         <i class="fa fa-edit" id="edit-button" aria-hidden="true"></i>
                     </a>
+                    {{else if can_manage_user}}
+                    <button type="button" id="modal-edit-user-button">
+                        <i class="fa fa-edit"></i>
```

Comment:
Agreed! That would be more accessible. How do you feel about adding a text element after the icon with a visually-hidden class for both of these? That way the icon gets hidden on screen readers, but the text is still visible. https://gomakethings.com/revisting-aria-label-versus-a-visually-hidden-class/

Reply:
The translation issue mentioned in that article is a non-issue, since we tag our `aria-label` strings for translation (and wrote a linter to require it). And that article doesn't provide clear examples of when `aria-label` isn't read by screenreaders. So I think we should stick with our current pattern of using `aria-label` as specified, but feel free to post that article in #frontend for discussion. I'd certainly be curious to learn more about it.